teuthology/task/install.py

```diff
@@ -225,24 +225,50 @@ def _update_rpm_package_list_and_install(ctx, remote, rpm, config):
     dist_release = gitbuilder.dist_release
     project = gitbuilder.project
     start_of_url = gitbuilder.base_url
-    proj_release = '{proj}-release-{release}.{dist_release}.noarch'.format(
-        proj=project, release=RELEASE, dist_release=dist_release)
+    if dist_release == 'opensuse':
+        proj_release = '{proj}-release-{release}.noarch'.format(
+            proj=project, release=RELEASE)
+    else:
+        proj_release = '{proj}-release-{release}.{dist_release}.noarch'.format(
+            proj=project, release=RELEASE, dist_release=dist_release)
     rpm_name = "{rpm_nm}.rpm".format(rpm_nm=proj_release)
     base_url = "{start_of_url}/noarch/{rpm_name}".format(
         start_of_url=start_of_url, rpm_name=rpm_name)
-    remote.run(args=['sudo', 'yum', '-y', 'install', base_url])
+    if dist_release == 'opensuse':
+        remote.run(args=[
+            'sudo', 'zypper', '--n', 'install', '--capability', rpm_name
+        ])
+    else:
+        remote.run(args=['sudo', 'yum', '-y', 'install', base_url])
 
-    uri = gitbuilder.uri_reference
-    _yum_fix_repo_priority(remote, project, uri)
-    _yum_fix_repo_host(remote, project)
-    _yum_set_check_obsoletes(remote)
+    if dist_release != 'opensuse':
+        uri = gitbuilder.uri_reference
+        _yum_fix_repo_priority(remote, project, uri)
+        _yum_fix_repo_host(remote, project)
+        _yum_set_check_obsoletes(remote)
 
-    remote.run(
-        args=[
-            'sudo', 'yum', 'clean', 'all',
-        ])
+    if dist_release == 'opensuse':
+        remote.run(
+            args=[
+                'sudo', 'zypper', 'clean', '-a',
+            ])
+    else:
+        remote.run(
+            args=[
+                'sudo', 'yum', 'clean', 'all',
+            ])
 
     ldir = _get_local_dir(config, remote)
+
+    if dist_release == 'opensuse':
+        pkg_mng_opts = '-n'
+        pkg_mng_cmd = 'zypper'
```

Comment:
Could you put ``pkg_mng_cmd`` before ``pkg_mng_opts`` please? Just for readability.